pom.xml

```diff
@@ -107,6 +115,12 @@
                     <systemPropertyVariables>
                         <allure.results.directory>${project.build.directory}/allure-results</allure.results.directory>
                     </systemPropertyVariables>
+                    <!--  Source for TestNG running     -->
+<!--                    <suiteXmlFiles>-->
+<!--                        <suiteXmlFile>-->
+<!--                            src\test\resources\testng.xml-->
+<!--                        </suiteXmlFile>-->
+<!--                    </suiteXmlFiles>-->
```

Comment:
why did you commite this part of the code?

Reply:
It belongs to TestNG. And as my main runner on the project is JUnit, so I decided to comment it. But it was unnecessary. Uncommented